server/routerlicious/packages/services-client/src/auth.ts

```diff
@@ -44,3 +50,28 @@ export function generateUser(): IUser {
 
     return randomUser;
 }
+
+/**
+ * Validates a JWT token to authorize routerlicious and returns decoded claims.
+ * An undefined return value indicates invalid claims.
+ */
+export function validateTokenClaims(
+    token: string,
+    documentId: string,
+    tenantId: string,
+    isTokenExpiryEnabled: boolean): ITokenClaims {
+    const claims = jwt.decode(token) as ITokenClaims;
+
+    const ONE_HOUR = 60 * 60; // 1 hour in seconds
```

Comment:
Let's add a config value for maxTokenLifetime instead. Set it to 1 hour as default in the config.
That way we can easily change the token lifetime if need be later.